<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="Button_clear">
              <property name="text">
               <string>AC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_sqrt">
              <property name="text">
               <string>âˆš</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_pow">
              <property name="text">
               <string>^</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_fact">
              <property name="text">
               <string>!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QPushButton" name="Button_1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="Button_4">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="Button_del">
              <property name="text">
               <string>Del</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="Button_dot">
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="Button_2">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="Button_6">
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="Button_0">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="Button_7">
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="Button_5">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="Button_9">
              <property name="text">
               <string>9</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="Button_8">
              <property name="text">
               <string>8</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="Button_3">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="Button_divide">
              <property name="text">
               <string>/</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_multiply">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_add">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_subtract">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Button_equal">
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculator">
    <property name="title">
     <string>Calculator</string>
    </property>
   </widget>
   <addaction name="menuCalculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
